js/src/lib/interfaces/Types.ts

```diff
@@ -578,6 +578,11 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 		color:        "blue",
 		hideInModels: true,
 	},
+	"text-to-video": {
+		name:     "Text to Video",
```

Comment:
<img width="778" alt="image" src="https://user-images.githubusercontent.com/3841370/226939162-7f5d9cd8-0970-444d-a9ac-15b61a2d72c4.png">
Maybe Text-to-Video to be consistent with Text-to-Image, or the opposite?

Reply:
yes i would add dashes too